dd-client/io_runner.sh

```diff
@@ -1,27 +1,31 @@
 #!/bin/bash
 
-#######################################################################################################################
-# Script Name   : io_runner.sh         									      		
-# Description   : Run dd profiles on the /datadir. 
-# Creation Data : 12/10/2018                                                                                          
-# Modifications : None											               		
-# Script Author : Sudarshan					      
-#######################################################################################################################
-
-TEST_SIZE="512k"
-TEST_COUNT="1000"
+blockSize=$(echo $BLOCK_SIZE)
+count=$(echo $COUNT)
+namespace=$(echo $NAMESPACE)
+mountPoint=$(echo $MOUNT_POINT)
+app_label=$(echo $APP_LABEL)
+retry_duration=$(echo $RETRY_DURATION)
+retry_count=$(echo $RETRY_COUNT)
 
 #Verify that the datadir used by the templates is mounted
-if ! df -h -P | grep -q datadir > /dev/null 2>&1; then
-    echo -e "datadir not mounted successfully, exiting \n"
-    exit 1
-fi
+count=0
```

Comment:
Are you sure this is the var name you want to use for the retry counter..consider bash vars have global scope by default? I see this being used to ingest an ENV var that represents the dd block count..Considering the pod is not available in the first trial (and is available after 2-3 sleep durations) -- this `count` will be initialized to 2 or 3, and we seem to be using this in the dd command after.